.config/ci/openssl.py

```diff
@@ -38,8 +38,11 @@
 """
 
 # Copy and edit
-with open(OPENSSL_CONFIG, 'rb') as fd:
-    DATA = fd.read()
+try:
+    with open(OPENSSL_CONFIG, 'rb') as fd:
+        DATA = fd.read()
+except FileNotFoundError:
+    DATA = b""
```

Comment:
It doesn't make sense to continue if this fails.
```suggestion
    print("WARNING: No OpenSSL config file found.", file=sys.stderr)
    sys.exit(1)
```

+ `import sys`

Reply:
Why is that? As we are providing our own openssl config file, everything work as expected.

Reply:
Oh you're right. I was thinking it made no sense to provide it if for instance libressl was used, but you're right there could be a case with no openssl config (or we couldn't find it).